Makefile

```diff
@@ -71,6 +71,10 @@ endif
 .PHONY: all
 all: bcachefs
 
+.PHONY: debug
```

Comment:
Adding a debug target is a good idea but it should replace the "make D=1 interface"

Reply:
Are you sure? I mean, adding Werror is quite a restrictive parameter. I'm absolutely a fan of adding `-DCONFIG_BCACHEFS_DEBUG=y -DCONFIG_VALGRIND=y`
but do you want the Werror as well?

Reply:
Yes - because warnings don't generally get fixed otherwise, and 'make debug' is really only for developers.

Reply:
commit 6d8cd43a95733cfacd15a52ba7459b4c5ff2410b should fix this